www/base/src/app/builders/builds/build.controller.coffee

```diff
@@ -72,7 +72,7 @@ class Build extends Controller
         data = dataService.open($scope)
         data.getBuilders(builderid).then (builders) ->
             $scope.builder = builder = builders[0]
-            builder.getBuilds(number__lt: buildnumber + 2, limit: 3, order: '-number').then (builds) ->
+            builder.loadBuilds(number__lt: buildnumber + 2, limit: 3, order: '-number').then (builds) ->
```

Comment:
does the page switching works that way? this might be another bug, because the builder losts the reference to builds and can't close its connection